resources/views/layouts/app.blade.php

```diff
@@ -16,112 +16,115 @@
+                        Mon code PIN
                     </a>
                 </li>
-                <li>
-                    <a href="https://twitter.com/LgHackerSpace" title="Our Twitter feed">
-                        <i class="fa fa-twitter"></i>
-                        Twitter
-                    </a>
+                <li class="{{ active('services') }}">
+                    <a href="{{ route('services') }}">Mes Services</a>
+                </li>
+                <li class="{{ active('app.rfid') }}">
+                    <a href="{{ route('app.rfid') }}">Cartes RFID</a>
+                </li>
+                @endpermission
+            </ul>
+            <ul class="nav navbar-nav navbar-right">
+                @permission(('manage-members'))
+                <li class="{{ active('admin.index') }}">
+                    <a href="{{ route('admin.index') }}">Admin</a>
                 </li>
+                @endpermission
                 <li>
-                    <a href="https://chat.lghs.be" title="Our Chat">
-                        <i class="fa fa-comment"></i>
-                        Chat
+                    <a href="{{ url('/logout') }}" title="Déconnectez-vous">
+                        <i class="fa fa-sign-out"></i>
+                        Déconnexion
+                        ({{{ isset(Auth::user()->name) ? Auth::user()->name : Auth::user()->email }}})
                     </a>
                 </li>
             </ul>
-        </footer>
-    </div>
+        </div><!-- /.navbar-collapse -->
+    </nav>
+
+    <main id="content">
+        <div class="container">
+            @yield('content')
+        </div>
+    </main>
+
+    <footer id="page-footer">
+        <p>
+            MAIL: ping[at]lghs.be
+        </p>
+        <p>
+            IRC: #LgHS @ Freenode
+        </p>
+        <ul>
+            <li>
+                <a href="https://github.com/LgHS" title="Our Github repos">
+                    <i class="fa fa-github"></i>
+                    Github
+                </a>
+            </li>
+            <li>
+                <a href="https://www.facebook.com/liegehackerspace/" title="Our Facebook page">
+                    <i class="fa fa-facebook"></i>
+                    Facebook
+                </a>
+            </li>
+            <li>
+                <a href="https://twitter.com/LgHackerSpace" title="Our Twitter feed">
+                    <i class="fa fa-twitter"></i>
+                    Twitter
+                </a>
+            </li>
+            <li>
+                <a href="https://chat.lghs.be" title="Our Chat">
+                    <i class="fa fa-comment"></i>
+                    Chat
+                </a>
+            </li>
+        </ul>
+    </footer>
+</div>
 
-    <script>
-        window.Laravel = { csrfToken: '{{ csrf_token() }}' };
-    </script>
-    <script type="text/javascript" src="{{asset('/js/app.js')}}"></script>
+<script>
+  window.Laravel = {csrfToken: '{{ csrf_token() }}'};
```

Comment:
I would freeze it using `Object.freeze` just to be extra pathologically paranoid.